src/main/docker/jenkins/jenkins.yml

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Jenkins base service
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    jenkins:
+        container_name: artemis-jenkins
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
+        build: .
+        user: root
+        volumes:
+            - artemis-jenkins-data:/var/jenkins_home
+            - ./jenkins-disable-csrf.groovy:/var/jenkins_home/init.groovy # Disable CSRF token
+            - ./master-key-only-for-testing.key:/var/jenkins_home/master.key # Preset master key to use pre-generated secrets
```

Comment:
I don't think that this should part of a production configuration

Reply:
Like the `atlassian.yml` docker files this isn't a hardened configuration, and its main intention was and still is for developers to get up and running faster. We could at some point of course introduce hardened overrides and separate the compose files in prod and dev variants.

Reply:
Perfect. As far as these configuration are only for development, everything is fine.